buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/ClosureUtil.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.gradle.plugins;
+
+import groovy.lang.Closure;
+import java.util.function.Consumer;
+
+public final class ClosureUtil {
```

Comment:
Can You help understand what this does?

Reply:
this is to express the following in java(this api is not java-friendly):
```groovy
android.testOptions.unitTests.all {
 // this is a groovy closure that is hard to implement in java, hence this util class
}
```